app/lib/submission_builder/ty2024/states/nj/documents/nj2450.rb

```diff
@@ -21,13 +21,13 @@ def document
                   xml.Body do
                     column_a = w2.box14_ui_wf_swf&.positive? ? w2.box14_ui_wf_swf : w2.box14_ui_hc_wd
```

Comment:
[pebble] i think this would want to use `get_column_a` to follow the same pattern as the others, right? (and I am on a semi-quest to DRY box 14 as much as possible)

Reply:
Unused variable, whoops!